src/core/CAG.js

```diff
@@ -101,6 +102,12 @@ CAG.prototype = {
     return fromSides(newsides)
   },
 
+  // ALIAS !
+  center: function (axes) {
+    axes = [axes[0],axes[1],false] // just be sure
```

Comment:
I do not quite understand this one ?  was it what the old version was doing ? if yes, wow , weird

Reply:
Hmm... good point. I’ll just pass the given argument through.